tests/data/PortalState.ts

```diff
@@ -57,6 +57,9 @@ export class PortalState {
     for (let u = 0; u < users.length; u += 1) {
       const user = users[u];
       user.email = user.email.toLowerCase();
+      if (user.points) user.credits = user.points * 100;
+      else if (user.credits) user.points = Math.floor(user.credits / 100);
```

Comment:
What's this line for? Why would a user have credits but not points?

Reply:
Ah I see. Maybe set `points: 50` instead in the test?

Reply:
Yeah, I have it in case a developer sets credits in a test and doesn't set points. I think it may cause silent issues if the points <-> credits invariant is broken when credits are set without points, so I personally think it's good to have this assignment to save debugging time. That way we can also set credits at the same scale as the cost of an item option instead of having to calculate the scale each time (even more beneficial if the scale gets changed in the future).

Reply:
👍🏽